pandas/_libs/tslibs/frequencies.pxd

```diff
@@ -1,4 +1,10 @@
 # -*- coding: utf-8 -*-
 # cython: profile=False
 
+cpdef object _get_rule_month(object source, object default=*)
+
```

Comment:
are these used anywhere *but* in period.pyx (for cython usage)?

Reply:
There's a duplicate of it in tslibs.parsing that should be replaced with a cimport.

Reply:
(things will be easier on my end if we can clean those up in a follow-up; im guessing you'll also suggest de-privatizing _get_rule_month)

Reply:
that's fine, pls add to the list.